Create lwm2m-agent-1021-10-18-incorrect-alarm-triggering-for-existing-lw-m-2-m-objects.md (#3464)
SoftwareAG/c8y-docs: content/change-logs/device-management/lwm2m-agent-1021-10-18-lwm2m-improved-logging-behavior.md

---

> date: ""

Reviewer: ```suggestion
date: 2025-06-19
```

> date: 2025-06-19

Reviewer 2: ```suggestion
date: 
```

Reviewer: This is already in zone-1, therefore I added the date

> title: Removed Redundant "LWM2M Device Send" Event in VERBOSE Logging for Missing Resource DDFs

Reviewer 2: ```suggestion
title: Removed Redundant "LWM2M Device Send" event in VERBOSE logging
```

product_area: Device management & connectivity
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
ticket: DM-4752
version: 1021.10.18
---

> The redundant "LWM2M Device Send" events was logged alongside the "Received Message" event, both containing identical information. This update removes the duplicate "LWM2M Device Send" event, only the "Received Message" event will now log the resource data received from the LWM2M Service in VERBOSE logging level.

Reviewer 2: ```suggestion
The "LWM2M Device Send" event was logged alongside the "Received Message" event, although both events contained identical information. This update removes the redundant "LWM2M Device Send" event. Only the "Received Message" event now logs the resource data received from the LWM2M service in the VERBOSE logging level.
```
